app/decorators/article_decorator.rb

```diff
@@ -13,4 +13,10 @@ def author
   def sorted_comments
     object.comments.sorted_by_created_at
   end
+
+  def rating
+    return "Nobody has rated on this article yet" unless object.ratings.any?
```

Comment:
Also, using locales is good practice
```suggestion
    return I18n.t("some.locale") unless object.ratings.any?
```